README.md

```diff
@@ -1,94 +1,127 @@
+## (Optional) Step 4: Additional configurations
+<details><summary>Expand for configurations</summary>
+    
+### Change the build schema
+By default, this package builds the Snapchat Ads staging models within a schema titled (`<target_schema>` + `_stg_snapchat_ads`) and your Snapchat Ads modeling models within a schema titled (`<target_schema>` + `_snapchat_ads`) in your destination. If this is not where you would like your Snapchat Ads data to be written to, add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
 models:
-    snapchat_ads:
-        +schema: my_new_schema_name # leave blank for just the target_schema
     snapchat_ads_source:
-        +schema: my_new_schema_name # leave blank for just the target_schema
+      +schema: my_new_schema_name # leave blank for just the target_schema
+    snapchat_ads:
+      +schema: my_new_schema_name # leave blank for just the target_schema
 ```
+    
+</details>
+
+## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
+<details><summary>Expand for details</summary>
+<br>
+    
+Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
+</details>
+
+# 🔍 Does this package have dependencies?
+This dbt package is dependent on the following dbt packages. Please be aware that these dependencies are installed by default within this package. For more information on the following packages, refer to the [dbt hub](https://hub.getdbt.com/) site.
+> IMPORTANT: If you have any of these dependent packages in your own `packages.yml` file, we highly recommend that you remove them from your root `packages.yml` to avoid package version conflicts.
+    
+```yml
+packages:
+    - package: fivetran/snapchat_ads_source
+      version: [">=0.4.0", "<0.5.0"]
 
-## Contributions
-
-Don't see a model or specific metric you would have liked to be included? Notice any bugs when installing and running the package? If so, we highly encourage and welcome contributions to this package! 
-Please create issues or open PRs against `master`. See [the Discourse post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) for information on how to contribute to a package.
-
-## Database Support
+    - package: fivetran/fivetran_utils
+      version: [">=0.3.0", "<0.4.0"]
 
-This package has been tested on BigQuery, Snowflake, Redshift, Postgres, and Databricks.
+    - package: dbt-labs/dbt_utils
+      version: [">=0.8.0", "<0.9.0"]
+```
+# 🙌 How is this package maintained and can I contribute?
```

Comment:
We should add a reference to the DECISIONLOG.md in this section. See how we did it [here](https://github.com/fivetran/dbt_google_ads/tree/feature/v2_updates#opinionated-decisions)